Runtime/Scripts/RGBotSpawnManager.cs

```diff
@@ -280,7 +280,15 @@ protected internal void CallSeatBot(BotInformation botToSpawn)
             {
                 // First, allow the developer to configure any frontend/backend for a player about to spawn,
                 // such as character selection.
-                SeatBot(botToSpawn);
+                try
+                {
+                    SeatBot(botToSpawn);
+                }
+                catch (Exception ex)
+                {
+                    RGDebug.LogError($"Error seating bot - {ex}");
```

Comment:
Should we also just return here or raise an exception? Not sure if we want to continue getting the bot ready if the seating failed.

Reply:
removed